repos/spack_repo/builtin/packages/libmesh/package.py

```diff
@@ -224,8 +226,8 @@ def configure_args(self):
             options.append("--enable-netcdf=no")
 
         if self.spec.satisfies("+vtk"):
-            options.append("--enable-vtk")
-            options.append("--with-vtk=%s" % self.spec["vtk"].prefix)
+            options.append("--enable-vtk-required")
+            options.append("VTK_DIR=%s" % self.spec["vtk"].prefix)
```

Comment:
@capitalaslash, you should add a justification why this change is needed to the PR or the code.